test/utils/domEvent.js

```diff
@@ -0,0 +1,46 @@
+import simulant from 'simulant'
+
+// ----------------------------------------
+// Simulate DOM Events on real DOM nodes
+// ----------------------------------------
+
+/**
+ * Generic method for dispatching an event on a DOM node.
+ * @param {String|Object} node A querySelector string or DOM node.
+ * @param {String} eventType A DOMString
+ * @param {Object} [data] Additional event data.
+ * @returns {Object} The event
+ */
+export const fire = (node, eventType, data = {}) => {
+  const DOMNode = typeof node === 'string' ? document.querySelector(node) : node
+  // TODO remove hack once merged: https://github.com/Rich-Harris/simulant/pull/11
+  // simulant has issues
+  //   - throws if params is not defined with 'keydown'
+  //   - does not extend with 'code' param, which we need for 'keydown'
+  // to solve these we:
+  //   - use an empty params object
+  //   - assign our own params
+
+  const DONT_USE_THESE_PARAMS = {}
+  const event = simulant(eventType, DONT_USE_THESE_PARAMS)
```

Comment:
Aye, intense for exactly that reason.  The bug I fixed in `simulant` was centered around passing in event data in that arg.  Wanted to ensure no one ever put data in that object, it had to remain empty.

With the bug fixed merged and shipped, I can remove this now.